pkg/ddevapp/app_compose_template.yaml

```diff
@@ -203,6 +204,7 @@ services:
     - TZ={{ .Timezone }}
     - USER={{ .Username }}
     - VIRTUAL_HOST=${DDEV_HOSTNAME}
+    - YARN_CACHE_FOLDER =/mnt/ddev-global-cache/yarn/${DDEV_PROJECT}-web
```

Comment:
Not trying to be pedantic, but there's a space before the `=`, I am 99% sure this is nothing, but thought of at least pointing it out.

Reply:
Try a github suggestion here please, your suggestion is correct :)

Reply:
```suggestion
    - YARN_CACHE_FOLDER=/mnt/ddev-global-cache/yarn/${DDEV_PROJECT}-web
```

Reply:
Thanks! Makes things so much easier!

You should now be able to `git pull` in your gitpod instance and `make` and test this with all your catches.

I'm sure embarrassed  that I fat-fingered 50% of the things you told me to do :)